replication.tf

```
@@ -28,7 +28,7 @@ data "aws_iam_policy_document" "replication_sts" {
 resource "aws_iam_policy" "replication" {
   count = local.replication_enabled ? 1 : 0
 
-  name   = format("%s-replication", module.this.id)
+  name   = "${aws_s3_bucket.default[0].id}-replication"
```

See https://github.com/cloudposse/terraform-aws-s3-bucket/pull/250/files#r1765084006

```suggestion
  name   = format("%s-replication, local.bucket_name)
```